src/componentes/Cadastro/Cadastro.jsx

```
@@ -1,200 +1,207 @@
+			cadastro_ip: {
+				municipio: municipio,
+				cargo: cargo,
+				telefone: telefone,
+				whatsapp: whatsapp,
+				mail: mail,
+			},
+		},
+	];
+	return (
+		<>
+			{etapa == 1 && (
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)